src/raftstore/coprocessor/region_snapshot.rs

```diff
@@ -225,7 +225,7 @@ mod tests {
     type DataSet = Vec<(Vec<u8>, Vec<u8>)>;
 
     fn new_temp_engine(path: &TempDir) -> Arc<DB> {
-        new_engine(path.path().to_str().unwrap(), DEFAULT_CFS).unwrap()
+        Arc::new(rocksdb::new_engine(path.path().to_str().unwrap(), DEFAULT_CFS).unwrap())
```

Comment:
I think there is not much difference, the return value of this function is a `rocksdb::DB` anyway.